source/extensions/extensions_build_config.bzl

```diff
@@ -131,6 +131,7 @@ EXTENSIONS = {
     "envoy.filters.network.ext_authz":                  "//source/extensions/filters/network/ext_authz:config",
     "envoy.filters.network.http_connection_manager":    "//source/extensions/filters/network/http_connection_manager:config",
     "envoy.filters.network.kafka_broker":               "//source/extensions/filters/network/kafka:kafka_broker_config_lib",
+    "envoy.filters.network.connection_limit":           "//source/extensions/filters/network/connection_limit:config",
```

Comment:
Can it be that this group is not sorted properly (`envoy.filters.network.connection_limit` should go after `envoy.filters.network.client_ssl_auth`)?

Reply:
Turn out it's due to missing metadata in the `extensions_metadata.yaml`, fixed it.

Reply:
Still it's better to keep the order consistent.